test/Buddy/functional/config/manticore.conf

```diff
@@ -0,0 +1,15 @@
+common {
```

Comment:
This file is the same that we used in branch that I made with kill tests.

We should think about using a single file for the manticore test. So maybe we should 

I have merged that branch into main, so I guess we can arrange single place for configs in root test dir (that would be better, when configs are in same place)

Reply:
Ok, I'll do rebase from main.